.ci-operator.yaml

```
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.18-openshift-4.11
+  tag: rhel-8-release-golang-1.19-openshift-4.11
```

You'll need to bump this to 4.13 as well
```suggestion
  tag: rhel-8-release-golang-1.19-openshift-4.13
```